plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-item-template/frontend.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * External dependencies
+ */
+import { store } from '@woocommerce/interactivity';
```

Comment:
Heads-up that we need to [change these imports](https://github.com/woocommerce/woocommerce/pull/55949/files) to `@wordpress/interactivity`:

```suggestion
import { store } from '@wordpress/interactivity';
```